signal SIGSEGV: segmentation violation on  jx boot command in AKS

### Summary 
I get a SIGSEGV: segmentation violation on  `jx boot` command going against AKS cluster.

### Steps to reproduce the behavior
ran `jx boot` once
it gave me the error
`error: : unable to parse enttestaks05 as <project id>_<zone>_<cluster name>`

after running jx boot once and having it clone the jx project, and failing 
I ran `jx compliance run` and cluster had no failures.

i update the file `jx-requirements.yml` and set cluster.provider to aks as such
```
cluster:
  clusterName: enttestaks05
  environmentGitOwner: hectoralicea
  gitKind: github
  gitName: github
  gitServer: https://github.com
  namespace: jx
  provider: aks
```
ran `jx boot` again and got the SIGSEGV after some time.

### Expected behavior
startup jenkins correctly.

The work around was to use the deprecated command `jx install` which worked.

### Actual behavior
```
STEP: create-helm-values command: /bin/sh -c jx step create values --name parameters in dir: /Users/hector/acme/jenkins-x/jenkins-x-boot-config/env

defaulting to secret storage scheme local found from requirements file at /Users/hector/acme/jenkins-x/jenkins-x-boot-config/jx-requirements.yml
defaulting to secret base path to the cluster name enttestaks05 found from requirements file at /Users/hector/acme/jenkins-x/jenkins-x-boot-config/jx-requirements.yml
generated schema file /Users/hector/acme/jenkins-x/jenkins-x-boot-config/env/parameters.schema.json from template /Users/hector/acme/jenkins-x/jenkins-x-boot-config/env/parameters.tmpl.schema.json
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x31e3ebb]

goroutine 1 [running]:
github.com/jenkins-x/jx/pkg/cmd/step/create.(*StepCreateValuesOptions).verifyRegistryConfig(0xc0002c9440, 0xc00086f880, 0xc00094c000, 0x49, 0x6601500, 0xc0007ac320, 0x0, 0x4ebd560)
	/workspace/source/pkg/cmd/step/create/step_create_values.go:266 +0x24b
github.com/jenkins-x/jx/pkg/cmd/step/create.(*StepCreateValuesOptions).Run(0xc0002c9440, 0x0, 0x0)
	/workspace/source/pkg/cmd/step/create/step_create_values.go:179 +0x746
github.com/jenkins-x/jx/pkg/cmd/step/create.NewCmdStepCreateValues.func1(0xc000878280, 0xc00045c520, 0x0, 0x2)
	/workspace/source/pkg/cmd/step/create/step_create_values.go:102 +0x6f
github.com/spf13/cobra.(*Command).execute(0xc000878280, 0xc00045c500, 0x2, 0x2, 0xc000878280, 0xc00045c500)
	/workspace/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0xc000349b80, 0xc000364500, 0x65b3c40, 0xc0000c0000)
	/workspace/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fc
github.com/spf13/cobra.(*Command).Execute(...)
	/workspace/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/jenkins-x/jx/cmd/jx/app.Run(0x0, 0x0, 0x0, 0xc0006e1f88, 0xc00005e118)
	/workspace/source/cmd/jx/app/jx.go:19 +0x185
main.main()
	/workspace/source/cmd/jx/jx.go:11 +0x32
error: failed to interpret pipeline file jenkins-x.yml: failed to run '/bin/sh -c jx step create values --name parameters' command in directory 'env', output: ''
```

### Jx version

The output of `jx version` is:

```
MacBook-Air:jenkins-x-boot-config hector$ jx version
NAME               VERSION
jx                 2.0.1094
jenkins x platform 2.0.1795
Kubernetes cluster v1.14.8
kubectl            v1.16.3
helm client        Client: v2.16.1+gbbdfe5e
git                2.24.0
Operating System   Mac OS X 10.14.6 build 18G1012
```

### Jenkins type

<!--
Select which installation type are you using. 
-->
- [ ] Serverless Jenkins X Pipelines (Tekton + Prow)
- [X] Classic Jenkins

### Kubernetes cluster

<!--
What kind of Kubernetes cluster are you using & how did you create it?
-->
AKS, created with terraform.  `kubectl get ns` commands works fine
### Operating system / Environment

<!--
In which environment are you running the jx CLI?
-->
MacOS

--- a/pkg/cmd/step/create/step_create_values.go
+++ b/pkg/cmd/step/create/step_create_values.go
@@ -181,7 +181,6 @@ func (o *StepCreateValuesOptions) Run() error {
 		return err
 	}
 
-	fmt.Println()
 	err = o.CreateValuesFile(secretURLClient)
 	if err != nil {
 		return errors.WithStack(err)
@@ -197,7 +196,6 @@ func (o *StepCreateValuesOptions) Run() error {
 		return errors.Wrap(err, "error creating the secret for local secret scheme")
 	}
 
-	fmt.Println()
 	return nil
 }
 
@@ -263,6 +261,9 @@ func (o *StepCreateValuesOptions) verifyRegistryConfig(requirements *config.Requ
 				return errors.Wrap(err, "getting cluster from Azure")
 			}
 			registryID := ""
+			if requirements.Cluster.AzureConfig == nil {
+				return errors.New("no azure registry subscription specified in 'jx-requirements.yml' at cluster.azure")
+			}
 			azureRegistrySubscription := requirements.Cluster.AzureConfig.RegistrySubscription
 			if azureRegistrySubscription == "" {
 				log.Logger().Info("no azure registry subscription is specified in 'jx-requirements.yml' at cluster.azure.registrySubscription")

